src/lib.rs

```diff
@@ -66,17 +66,6 @@ pub mod __private {
     pub use crate::vendor::thiserror::aserror::AsDynError;
 }
 
-/// Module containing macro definitions only, without corresponding traits.
-///
-/// Use it in your import paths, if you don't want to import traits, but only macros.
-pub mod derive {
-    // This can be unused if no feature is enabled. We already error in that case, but this warning
-    // distracts from that error. So we suppress the warning.
-    #[allow(unused_imports)]
-    #[doc(inline)]
-    pub use derive_more_impl::*;
-}
```

Comment:
Should we be removing this module? I feel like it's a somewhat unnecessary backwards compatibility break. At the very least this removal should be documented in the changelog.